lib/lens/allocations_data.rb

```diff
@@ -0,0 +1,46 @@
+module Lens
+  if RUBY_ENGINE == 'ruby' && RUBY_VERSION >= '1.9'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.